easylistportuguese/easylistportuguese_specific_hide.txt

```diff
@@ -256,4 +256,9 @@ tecnoblog.net##div[id^="fullbanner-container"]
 tecnoblog.net##div[id^="sidebanner_single"] 
 tecnoblog.net##div[id^="company-header-ad"]
 tecnoblog.net##section[id^="ad-superbanner-"]
-br.ccm.net,saude.ccm.net##div[class^="ads--insertor-"]
\ No newline at end of file
+br.ccm.net,saude.ccm.net##div[class^="ads--insertor-"]
+cenariomt.com.br##a[href^="https://www.inexa.com.br"]
+cenariomt.com.br##a[href^="http://www.agrobaggio.com.br"]
+cenariomt.com.br##a[data-img-url^="https://www.cenariomt.com.br/wp-content/uploads/2021/07/amazoniajuho2021.gif"]
+cenariomt.com.br##a[href^="http://encurtador.com.br"]
+cenariomt.com.br##a[href^="https://www.modelocontabilidade.com"]
```

Comment:
These filters also belong to a different PR
https://github.com/easylist/easylistportuguese/pull/76